src/content/docs/configuration/providers.mdx

```diff
@@ -20,7 +20,7 @@ Daytona abstracts the management and deployment of Workspaces into 2 related con
     Daytona allows you to configure new and existing Targets, and choose between them at Workspace creation time.
 
 Daytona's architecture decentralizes the role of a Provider into a separate service.
-The [Daytona Server](../reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
+The [Daytona Server](/reference/server) communicates with Providers to execute operations relating to Workspace creation and lifecycle management.
```

Comment:
Please test **ALL** the links you changed.

Reply:
Resolved already. See properly.

Reply:
https://github.com/user-attachments/assets/09556f70-1e6d-4561-93b1-32a12e930a9e

You're really testing me here 😄 

I'll close the PR if you request a new review without testing **ALL** the links you changed.

Reply:
Ok 👌 I will check **All** the links 😃 and then make a review